docs/architecture.md

```diff
@@ -44,11 +44,7 @@ For this reason, the chunks storage consists of:
   * [Google Cloud Storage](https://cloud.google.com/storage/)
   * [Microsoft Azure Storage](https://azure.microsoft.com/en-us/services/storage/)
 
-Internally, the access to the chunks storage relies on a unified interface called "chunks store". Unlike other Cortex components, the chunk store is not a separate service, but rather a library embedded in the services that need to access the long-term storage: [ingester](#ingester), [querier](#querier) and [ruler](#ruler).
```

Comment:
This whole content has not been deleted, but moved (in a different form) under the "Chunks storage" section.